CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 # Changelog
 
-## 2.8.4 - 2022-11-11
-- [Bugfix]: fix: close virtualaccount end point [#310]
-- [Bugfix]: fix: passing `partial_payment` boolean issue when creating order [#311]
--  Added Issue and feature templates
+## 2.8.4 - 2022-11-21
+
+- [#310](https://github.com/razorpay/razorpay-node/pull/310) [`3e6daf3`](https://github.com/razorpay/razorpay-node/commit/3e6daf3c555f62eb23660a54eaae756e395ea3b6) : Thanks [@ankitdas13 ](https://github.com/snitin315)! - Fixed `virtualAccount.close` function implementation
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏ @ankitdas13 Please update Thanks [@ankitdas13 ](https://github.com/snitin315)! with your github account link 😅

Reply:
@sidag95 sorry i didnt saw that 😅 . i have updated the account link

Reply:
@ankitdas13 Last thing, why an extra space after `@ankitdas13`?

Reply:
@sidag95 
**With space** [@ankitdas13 ](https://github.com/ankitdas13)!
**Without space** [@ankitdas13](https://github.com/ankitdas13)!

Should i remove space ?

Reply:
Yup, lets remove it